package.json

```diff
@@ -25,7 +25,7 @@
     "generate:devo-angularjs-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o dist/typescript -g typescript-angularjs && tsc --project tsconfig.json && npm run clean-source",
     "generate:devo-angular-ci": "npm run download:devo && gulp codegen && java -jar /opt/openapi-generator-630/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i api/otr-service.json -o dist/angular -g typescript-angular",
     "generate:devo-angular": "npm run download:devo && gulp codegen &&  openapi-generator-cli version-manager set 6.3.0 && openapi-generator-cli generate -i api/otr-service.json -o dist/angular -g typescript-angular",
-    "generate:devo-typescript-fetch": "npm run download:devo && gulp codegen &&  openapi-generator-cli version-manager set 4.3.1 && openapi-generator-cli generate -i api/otr-service.json -o dist/typescript-fetch -g typescript-fetch --additional-properties=typescriptThreePlus=true && tsc --project tsconfig.fetch.json && npm run clean-source",
+    "generate:devo-typescript-fetch": "rm -r dist/typescript-fetch || npm run download:devo &&  openapi-generator-cli version-manager set 4.3.1 && openapi-generator-cli generate -i api/otr-service.json -o myclient/typescript-fetch -g typescript-fetch --additional-properties=typescriptThreePlus=true && tsc --project tsconfig.fetch.json  --outDir dist/typescript-fetch",
```

Comment:
nice. definitely worth adding to the devo-angularjs and devo-angularjs-ci npm tasks